packages/plugins/connection-manager/explorer/tree-items.ts

```diff
@@ -299,10 +299,15 @@ export class SidebarFunction extends SidebarAbstractItem<null> {
     this.tooltip = `${this.functionData.signature}(${this.functionData.args.join(',')}) => ${this.functionData.resultType || 'void'}`;
     this.value = `${this.functionData.signature}`;
     let args = [];
-    this.functionData.args.forEach((type, index) => {
-      args.push(`\${${index + 1}:${type}}`);
+    this.functionData.args.forEach((type, index) => {      
+      const [argName, argType] = type.split('=>');
```

Comment:
I am not sure if other dialects also use `=>` for parameter name and parameter value separator... but other separator could be added later if they exist.

Reply:
That's fine! Thanks!

I'll just update `if (argType !== null)` to `if (argType && argType.trim())`  because it as breaking on other dialects.

Besides that, we are good to go!

@mickeypearce great job!

Reply:
no problem. sorry, but does't string.trim() return new value and doesn't change the original? :)

Reply:
Nope, just if I reassign it like `argType = argType.trim()`.

Just trim returns a new string

Reply:
what does trim() in `if (argType && argType.trim())` do then?

Reply:
it's equivalent (not really but in this context it's valid) to this:

```javascript
if (argType) {
  if (argType.length > 0) {
    // do stuff
  }
}
```

or in a single line like that:
```javascript
if (argType !== null && typeof argType !== 'undefined' && argType.length > 0) {
  // do stuff
}
```

What was breaking for PostgreSQL was that `argType` was `undefined`, so I used a short hand to ensure argType exists.

This short hand is valid because `''` (empty string) in JS is false when casting to boolean.

Try it in the browser:
```javascript 
Boolean('') === false // this is true
```

Reply:
I get it now. Thanks for the detailed explanation!